cmd/node_agent/na/BUILD

```diff
@@ -4,29 +4,21 @@ go_library(
     name = "go_default_library",
     srcs = [
         "config.go",
-        "gcp.go",
-        "aws.go",
         "nafactory.go",
         "nodeagent.go",
-        "onprem.go",
         "util.go",
     ],
     visibility = ["//visibility:public"],
     deps = [
-        "//pkg/credential:go_default_library",
         "//pkg/pki:go_default_library",
         "//pkg/pki/ca:go_default_library",
-        "//pkg/util:go_default_library",
+        "//pkg/platform:go_default_library",
+        "//pkg/platform/mock:go_default_library",
```

Comment:
Does it need mock package? This will build mock client into the node agent binary.

Reply:
No, it should be in the test package. Thanks for catching this.